stix_shifter_modules/paloalto/configuration/config.json

```diff
@@ -0,0 +1,41 @@
+{
+    "connection": {
+        "type": {
+            "displayName": "Palo Alto Cortex XDR"
+        },
+        "host": {
+            "type": "text"
+        },
+        "help": {
+            "type": "link",
+            "default": "data-sources.html"
+        },
+        "options": {
+            "type": "fields",
+            "result_limit": {
```

Comment:
```
"result_limit": {
                "default": 1000,
                "max": 1000
            }
```

Reply:
just wondering why the max result  is`1000`?

Reply:
Since stream id options is implemented which in turn result_limit from default config will be used and this configuration will not be applicable anymore.